### PR TITLE
Hide Content for Workshops 1-6

### PR DESCRIPTION
**Pull request by nik-dange**

### CODE REVIEW

**Review by angela139 — APPROVED** [review]
LGTM!

**Review by khushijpatel — APPROVED** [review]
looks good

File: pages/hack-school/index.mdx [hunk 1]
```diff
@@ -12,5 +12,5 @@ We chose the MERN stack for its wide adoption in the development community, ease
 prerequisites for this course -- we've designed it so that you learn as you go through it!
 
 ## How do I use this website?
-Under the Hack School tab on the left, you'll find pages for every technology and concept we teach during Hack School, so you won't be missing out on any content. Each page has a sidebar on the right
-that you can use to quickly navigate through the content. We've included code samples and links to relevant documentation to help you as you work through our material!
+Under the Hack School tab on the left, you'll find pages for every technology and concept we teach during Hack School, so you won't be missing out on any content. We'll be releasing content after every workshop,
```
Comment (alexzhang1618) [inline pages/hack-school/index.mdx:15]: is it released after every workshop or before?

Comment (nik-dange) [reply pages/hack-school/index.mdx]: good catch, just before the workshop

**Pull request merged**